format/fit/testdata/settings.fqtest

```diff
@@ -0,0 +1,125 @@
+0x40|            40                                 |    @           |        header_type: "normal" (0) 0x44-0x44.1 (0.1)
+0x40|            40                                 |    @           |        message_type: "definition" (1) 0x44.1-0x44.2 (0.1)
+0x40|            40                                 |    @           |        has_developer_fields: false 0x44.2-0x44.3 (0.1)
+0x40|            40                                 |    @           |        reserved: false 0x44.3-0x44.4 (0.1)
+0x40|            40                                 |    @           |        local_message_type: 0 0x44.4-0x45 (0.4)
+    |                                               |                |      definition_message{}: 0x45-0x4d (8)
+0x40|               00                              |     .          |        reserved: 0 0x45-0x46 (1)
+0x40|                  01                           |      .         |        architecture: "big_endian" (1) 0x46-0x47 (1)
+0x40|                     00 04                     |       ..       |        global_message_number: "hrm_profile" (4) 0x47-0x49 (2)
+0x40|                           01                  |         .      |        fields: 1 0x49-0x4a (1)
+    |                                               |                |        field_definitions[0:1]: 0x4a-0x4d (3)
+    |                                               |                |          [0]{}: field_definition 0x4a-0x4d (3)
+0x40|                              01               |          .     |            field_definition_number: "hrm_ant_id" (1) 0x4a-0x4b (1)
+0x40|                                 02            |           .    |            size: 2 0x4b-0x4c (1)
+0x40|                                    8b         |            .   |            base_type: "uint16z" (139) 0x4c-0x4d (1)
+    |                                               |                |    [5]{}: data_record 0x4d-0x50 (3)
+    |                                               |                |      record_header{}: 0x4d-0x4e (1)
+0x40|                                       00      |             .  |        header_type: "normal" (0) 0x4d-0x4d.1 (0.1)
+0x40|                                       00      |             .  |        message_type: "data" (0) 0x4d.1-0x4d.2 (0.1)
+0x40|                                       00      |             .  |        has_developer_fields: false 0x4d.2-0x4d.3 (0.1)
+0x40|                                       00      |             .  |        reserved: false 0x4d.3-0x4d.4 (0.1)
+0x40|                                       00      |             .  |        local_message_type: 0 0x4d.4-0x4e (0.4)
+    |                                               |                |      data_message{}: 0x4e-0x50 (2)
+0x40|                                          00 64|              .d|        hrm_ant_id: 25600 0x4e-0x50 (2)
+0x50|39 50|                                         |9P|             |  crc: 20537 (valid) 0x50-0x52 (2)
```

Comment:
Super nitpick: In some future PR maybe this can be presented in hex? add a `scalar.UintHex` mapper. Maybe there are more fields more suitable in hex?

I've thought about generalising this for crc-ish fields somehow not sure how, would be nice to provide both calculated and expected value.